### PR TITLE
Fixed #27973 -- Fixed GeoJSON representation of LinearRing and custom GEOSGeometry subclasses.

### PR DESCRIPTION
**Pull request by sir-sigurd**
https://code.djangoproject.com/ticket/27973

### CODE REVIEW

File: django/contrib/gis/geos/linestring.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 class LineString(LinearGeometryMixin, GEOSGeometry):
     _init_func = capi.create_linestring
     _minlength = 2
+    _json_type = 'LineString'
```
Comment (charettes) [inline django/contrib/gis/geos/linestring.py]: Shouldn't this be overridden in `LinearRing` instead?

Comment (sir-sigurd) [inline django/contrib/gis/geos/linestring.py]: Thanks to your question I fixed it for custom subclasses too.

File: django/contrib/gis/geos/collections.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
 
 class GeometryCollection(GEOSGeometry):
     _typeid = 7
+    _json_type = 'GeometryCollection'
```
Comment (timgraham) [inline django/contrib/gis/geos/collections.py]: Do you want to alphabetize the order of the attributes? (looks like that's the current ordering)

File: django/contrib/gis/geos/point.py [hunk 3]
```diff
@@ -9,6 +9,7 @@
 class Point(GEOSGeometry):
     _minlength = 2
     _maxlength = 3
```
Comment (sir-sigurd) [inline django/contrib/gis/geos/point.py]: @timgraham 
Attributes are not ordered here, so I'm not sure how to proceed with your suggestion...

Comment (timgraham) [inline django/contrib/gis/geos/point.py]: Putting `_j` before `_m` seems close enough.

**Pull request merged**